docs/team/wjunjie01.md

```diff
@@ -1 +1,57 @@
 <!--- @@author wjunjie01 --->
+# Wong Jun Jie's - Project Portfolio Page
+
+## Overview
+StockPal is an inventory management application, designed to help small business owners like you to manage your stock-taking effectively.
+StockPal is a command line interface (CLI) application, which means that it is run using a text-based interface.
+
+## Summary of Contributions
+
+### Code contributed
+Here is the link to my code:
+[tP Code Dashboard](https://nus-cs2113-ay2324s2.github.io/tp-dashboard/?search=wjunjie01&breakdown=true&sort=groupTitle%20dsc&sortWithin=title&since=2024-02-23&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=EdmundTangg&tabRepo=AY2324S2-CS2113T-T09-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
+<br/>
+
+### Features implemented
+**1. List command Feature**
+[#37](https://github.com/AY2324S2-CS2113T-T09-3/tp/pull/37)
+
+What it does:
+This feature displays all products that are stored in the inventory.
+
+Justification:
+- This feature is the core of the application. <br>
+A user needs to be able to view all the products in the application to monitor the products' details, 
+and to make meaningful changes to them.
```

Comment:
Perhaps can add in the part that, the list feature will enable users to get the PID?